force-app/tests/Email Recipes/InboundEmailHandlerRecipes_tests.cls

```diff
@@ -0,0 +1,140 @@
+@isTest
+private class InboundEmailHandlerRecipes_tests {
+    static final String FROM_EMAIL = 'kjp@codefriar.com';
+
+    @TestSetup
+    static void makeData() {
+        TriggerHandler.bypass('AccountTriggerHandler');
+        TriggerHandler.bypass('MDTAccountTriggerHandler');
+        Account testAccount = (Account) TestFactory.createSObject(
+            new Account(Name = 'Testing Account'),
+            true
+        );
+        Contact testContact = (Contact) TestFactory.createSObject(
```

Comment:
You could drop the unused `testContact` variable since you're not doing anything with the return of `TestFactory.createSObject`.
I recommend that you add a comment about your intent is for `makeData` because I failed to see this line and I was wondering later on why you where running delete operations in your tests.

Reply:
@pozil,

The testFactory's second paramter (after the contact object) is `true` which is the testFactory flag for 'should I insert this?' So we need to keep this testFactory call in place, but I agree I should comment it. Doing so now.